jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/exceptions/exceptionhandling/ApiResponseExceptionHandler.java

```diff
@@ -42,9 +42,9 @@ public ResponseEntity<ApiErrorResponse> handleTypeMismatchException(TypeMismatch
 
     @ExceptionHandler(PersistenceException.class)
     public ResponseEntity<ApiErrorResponse> handlePersistenceException(PersistenceException ex) {
-        LOGGER.severe("Persistence exception occurred whilst saving a resource. " + ex.getMessage());
-        var msg = "There was an error persisting data.";
-        return createResponseEntity(msg, HttpStatus.BAD_REQUEST);
+        LOGGER.severe("Server Error: Unable to process the request. " + ex.getMessage());
+        var msg = "Server Error: Unable to process the request.";
```

Comment:
very minor, doesn't need to change. But this variable could also be used in the log message.